spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/SamplingZipkinSpanIterator.java

```diff
@@ -38,6 +41,11 @@
  * @since 1.0.0
  */
 final class SamplingZipkinSpanIterator implements Iterator<zipkin.Span> {
+	private static final List<String> ZIPKIN_ANNOTATIONS = Arrays.asList(
```

Comment:
basically, I'd probably just call this `ZIPKIN_START_EVENTS = Arrays.asList(CLIENT_SEND, SERVER_RECV);` effectively these are the events that indicate it is an RPC span, and also whether or not an _ADDR binary annotation should be logged.